index.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Apple</title>
+        <link rel="stylesheet" href="styles/reset.css">
+        <link rel="stylesheet" href="styles/style.css">
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        
+        <style>
+            @import url('https://fonts.googleapis.com/css?family=Roboto:100');
+            </style>
+    </head>
+    <body> 
+        <div class="wrapp-all">
+            <header class="elements first-item">       
+                <ul>
```

Comment:
Use a nav tag here instead of a ul tag,  you could even remove the header tag and just keep the nav